testbed/tests/widgets/test_detailedlist.py

```diff
@@ -233,14 +233,9 @@ async def test_refresh(widget, probe):
         pytest.skip("This backend doesn't support the refresh action")
 
     # Set a refresh handler that simulates a reload altering data.
-    async def add_row(event_widget, **kwargs):
+    def add_row(event_widget, **kwargs):
         assert event_widget == widget
         assert kwargs == {}
-
-        # Simulate a reload delay
-        await probe.redraw("Wait for simulated reload")
```

Comment:
This made the test unreliable in `--slow` mode, because the handler and the test function are running in separate asyncio tasks and there's no guarantee of which one will finish awaiting first.